test/unit/util/operator/test-operators-identity-project.cpp

```diff
@@ -0,0 +1,83 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
```

Comment:
What does the term "project" in filename and in test names refer to?

Reply:
the project1st and project2nd operator tests

Reply:
I still don't understand.

Reply:
@mdavis36 , do you mean like a projection?